package/crds/networking.digitalocean.crossplane.io_loadbalancers.yaml

```diff
@@ -82,83 +89,7 @@ spec:
                     items:
                       type: number
                     type: array
-                  dropletIdsRefs:
```

Comment:
Can you confirm that these deletions are correct?

I see that most `refs` are gone from the CRD, so I am not sure where they should go now, but it feels like a breaking change.

Reply:
Thanks for the hint - i know why and will update the PR